src/Engine/Component/Script.hpp

```diff
@@ -69,6 +69,7 @@ namespace Component {
             ENGINE_API void ClearPropertyMap();
 
         private:
+            Script(const Script & other) = delete;
 
             class Property {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') class 'Property' does not have a copy constructor which is recommended since the class contains a pointer to allocated memory. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AnoCopyConstructor)